src/app.tsx

```diff
@@ -1,6 +1,6 @@
 // lib imports
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import React from 'react';
```

Comment:
@wongwill86 I think that if React is not supporting ES Module flag we should probably stick to default import, this is really annoying because we will not be able to use named import as well, but unfortunately I couldn't find any good workaround ;/